cmd/tag/main.go

```diff
@@ -96,4 +97,6 @@ func printMetadata(m tag.Metadata) {
 	fmt.Printf(" Picture: %v\n", m.Picture())
 	fmt.Printf(" Lyrics: %v\n", m.Lyrics())
 	fmt.Printf(" Comment: %v\n", m.Comment())
+
+	ioutil.WriteFile("bla", m.Picture().Data, 0777)
```

Comment:
This make it in by mistake?

Reply:
Yeap! reverted, too fast with the git add -u